models/desired_lrp_test.go

```diff
@@ -1002,6 +1007,18 @@ var _ = Describe("DesiredLRP", func() {
 			}
 		})
 
+		It("fails when service binding files exceed 1MB", func() {
+			var exceedSize string
+			for range (1024 * 1024) + 100 {
+				exceedSize += "a"
+			}
+
+			var InvalidServiceBindingFiles = []*models.File{{Path: "/redis/username", Content: exceedSize}}
+			desiredLRP.VolumeMountedFiles = InvalidServiceBindingFiles
+
+			assertDesiredLRPValidationFailsWithMessage(desiredLRP, "volumeMountedFiles")
+		})
+
```

Comment:
This tests when one file is individually greater than the maxSize. 

❓ Can you also add a test for DesiredLRP when there are multiple volume mounted files, where each file is less than the maxSize, but combined they are greater than the maxSize?